.clang-tidy

```diff
@@ -20,3 +20,4 @@ CheckOptions:
   - { key: readability-identifier-naming.ParameterCase,       value: camelBack }
 HeaderFilterRegex: '.*'
 WarningsAsErrors: '*'
+UseColor: true
```

Comment:
If it doesn't break on the CI, I think it's okay to go with this for now, but it can cause issues on certain terminals.  
But that's really more an edge case. If someone reports troubles, we can still disable it again.

Reply:
I have no problem reverting this later if someone reports problems